public/app/features/alerting/unified/components/rule-editor/LabelsField.tsx

```diff
@@ -271,71 +261,75 @@ const LabelsField: FC<Props> = ({ dataSourceName }) => {
 
   return (
     <div>
-      <Label description="A set of default labels is automatically added. Add additional labels as required.">
-        <Stack gap={0.5} alignItems="center">
-          <Text variant="bodySmall" color="primary">
-            Labels
+      <div className={styles.labelsDescription}>
+        <Text element="h5">Labels</Text>
+        <Stack direction={'row'} gap={1}>
+          <Text variant="bodySmall" color="secondary">
+            Add labels to your rule to annotate your rules, ease searching, or route to a notification policy.
```

Comment:
We may want to revise the copy here, grammar seems a bit weird – any thoughts here @brendamuir?